quickstart/faqs.md

```diff
@@ -1,62 +1,70 @@
-# Mayastor FAQs
+# OpenEBS Mayastor FAQs
 
 ## How does it help to keep my data safe?
-Mayastor's storage engine, or "nexus" as we term it, provides synchronous mirroring functionality to enhance the durability of data at rest within whatever physical persistence layer is in use.  When volumes are provisioned which are configured for replication \(a user can control the count of active replicas which should be maintained, on a per StorageClass basis\), write I/O operations issued by an application to that volume are 'amplified' by the nexus and sent to all active replicas.  Only if every replica completes the write successfully on its own underlying block device will the I/O completion be acknowledged.  Otherwise, the I/O is failed and must be tried by the caller.  If a replica is determined to have faulted \(I/O cannot be serviced within the configured timeout period, or not without error\), the control plane will automatically take corrective action and remove it from the volume.  If spare capacity is available within another Mayastor pool, a new replica will be created as a replacement and automatically brought into syncrhonisation with the existing replicas.  The data path for a replicated volume is described in more detail [here](https://github.com/openebs/mayastor-docs/blob/master/reference/i-o-path-description.md#replicated-volume-io-path)
+Mayastor's approach is to as simply as possible create copies of data so that when a node is rescheduled, or lost, that otherwise would cause loss of the data on that node, instead the data is available elsewhere and is made available to the workload.  
+
+Mayastor's storage engine, or "nexus", provides synchronous mirroring functionality to enhance the durability of data at rest within whatever physical persistence layer is in use.  When volumes are provisioned which are configured for replication \(a user can control the count of active replicas which should be maintained, on a per StorageClass basis\), write I/O operations issued by an application to that volume are 'amplified' by the nexus and sent to all active replicas.  Only if every replica completes the write successfully on its own underlying block device will the I/O completion be acknowledged.  Otherwise, the I/O is failed and must be retried by the caller.  If a replica is determined to have faulted \(I/O cannot be serviced within the configured timeout period, or not without error\), the control plane will automatically take corrective action and remove it from the volume.  If spare capacity is available within another Mayastor pool, a new replica will be created as a replacement and automatically brought into syncrhonisation with the existing replicas.  In short, Mayastor adds availability and consistency for data through ensuring that a minimum set of replicas ara available and by recovering from outages to retain that minimum number of replicas.  

```

Comment:
spellcheck `syncrhonisation`

Reply:
spellcheck `minimum set of replicas **ara** available`

Reply:
It might be nice if we have shorter sentences.

Reply:
Why have we removed the link to the data path?  Does this add not add value?